app/src/layout/ThemeSwitcher/ThemeSwitcher.tsx

```diff
@@ -1,29 +1,30 @@
 import { useContext } from "react";
 import { HvBox, HvDropdown, HvThemeContext } from "@hitachivantara/uikit-core";
+import styled from "@emotion/styled";
+
+const StyledDropdown = styled(HvDropdown)({
+  width: 150,
+});
 
 export const ThemeSwitcher = () => {
   const { selectedTheme, selectedMode, colorModes, themes, changeTheme } =
     useContext(HvThemeContext);
 
   return (
     <HvBox sx={{ display: "flex", gap: 20 }}>
-      <HvDropdown
```

Comment:
The `HvDropdown` component is also used in the stories associated with the theming. Thus, we should also apply these fixes to the `.storybook/docs/Theme/Theming/ThemeSwitcher` and  `.storybook/docs/Theme/Theming/ThemeStructures` files.